src/store/ChartStore.js

```diff
@@ -1194,26 +1194,17 @@ class ChartStore {
             (document.webkitFullscreenElement && document.webkitFullscreenElement !== null) ||
             (document.mozFullScreenElement && document.mozFullScreenElement !== null) ||
             (document.msFullscreenElement && document.msFullscreenElement !== null);
+        const fullscreen_map = {
+            event: ['fullscreenchange', 'webkitfullscreenchange', 'mozfullscreenchange', 'MSFullscreenChange'],
+            element: ['fullscreenElement', 'webkitFullscreenElement', 'mozFullScreenElement', 'msFullscreenElement'],
```

Comment:
You can get rid of `event` and use `element` to check for `isInFullScreen`.

Reply:
Done. Thank you @matin-binary :)